polarrouteserver/route_api/views.py

```diff
@@ -128,9 +124,7 @@ def post(self, request):
             logger.info("Creating new vehicle:")
 
             # Create vehicle in database
-            vehicle = Vehicle.objects.create(
-                vessel_type=vessel_type, **vehicle_properties
-            )
+            vehicle = Vehicle.objects.create(**data)
```

Comment:
Nice. Hadn't spotted the unpacking here, I basically never remember to use this.